modules/navier_stokes/include/variables/BernoulliPressureVariable.h

```diff
@@ -27,23 +27,31 @@ class BernoulliPressureVariable : public INSFVPressureVariable, public ADFunctor
 
   static InputParameters validParams();
 
-  bool isExtrapolatedBoundaryFace(const FaceInfo & fi, const Elem * elem) const override;
+  bool isExtrapolatedBoundaryFace(const FaceInfo & fi,
+                                  const Elem * elem,
+                                  const Moose::TimeArg & time) const override;
 
   void initialSetup() override;
 
 protected:
-  bool isDirichletBoundaryFace(const FaceInfo & fi, const Elem * elem) const override;
+  bool isDirichletBoundaryFace(const FaceInfo & fi,
```

Comment:
Do we support time-dependent boundaries? When in one step it is a Dirichlet in the other it isn't?